api/src/main/java/eu/darkbot/api/managers/ConfigAPI.java

```diff
@@ -169,6 +170,15 @@ default Set<String> getChildren(ConfigSetting<?> root, String path) {
         return null;
     }
 
+    /**
+     * Change current config.
+     * Sample config selector:
+     *   @Option @Dropdown(options = ConfigAPI.class) String conf = "(none)";
```

Comment:
probably best to move this to the `ConfigNames` (or similar) class, maybe add an `@see ConfigNames` or similar

Reply:
Reverted